bioinfokit/visuz.py

```diff
@@ -631,7 +634,7 @@ def mhat(df="dataframe", chr=None, pv=None, log_scale=True, color=None, dim=(6,4
             df['tpval'] = df[pv]
         # df = df.sort_values(chr)
         # if the column contains numeric strings
-        df = df.loc[pd.to_numeric(df[chr], errors='coerce').sort_values().index]
+        df = df.loc[pd.to_numeric(df[chr], errors='ignore').sort_values().index]
```

Comment:
'coerce' will produce nan if not convertible, while ignore will maintain the input. This change from makes the column still sortable even if it cannot be converted to numbers. If the column isn't sorted, it will result in color problem in the plot.